docs/node-operators/troubleshooting.mdx

```diff
@@ -87,15 +87,15 @@ See the requirements for the current network [here](./block-producer-node/gettin
 
 ### I would like the mina service to start on boot?
 
-If you use `systemd` to manage the Mina daemon then it will not automatically restart on a reboot of the machine. You can manually run the service via `systemctl --user start mina` or run `systemctl --user enable mina` to have the service automatically start on boot.
+If you use `systemd` to manage the Mina daemon then it will not automatically restart on a reboot of the machine. You can manually run the service with `start` via `systemctl --user start mina` or start the service automatically at boot with `enable` via `systemctl --user enable mina`.
```

Comment:
```suggestion
If you use `systemd` to manage the Mina daemon then it will not automatically restart on a reboot of the machine. You can manually run the service with `start` via `systemctl --user start mina` or start the service automatically at boot with `enable` by using `systemctl --user enable mina`.
```

Reply:
avoid Latin, instead use "by using"

Reply:
I via is an English word, but happy with your suggestion.